sycl/test/check_device_code/native_cpu/sycl-native-cpu-libclc.cpp

```diff
@@ -1,5 +1,8 @@
 // TODO: move this to clang/Driver once Native CPU is enabled in CI
 // REQUIRES: native_cpu
+// TODO: currently disabled on Windows, but could work on Windows if Linux
+// libraries are found
+// UNSUPPORTED: (windows)
```

Comment:
For good measure, could you please open a Github issue and add a link in a comment here?

Reply:
done! https://github.com/intel/llvm/issues/14500